### DISCUSSION TITLE
How to apply changes at end of using ChatGPTEditWithInstructions

### DISCUSSION THREAD

**Discussion by nbossard** [jackMort/ChatGPT.nvim#323] [Q&A]
Hello, 
I have selected some code and edited it using ChatGPTEditWithInstructions, but I can't find how to apply the changes to my source code when happy with it
Thanks, Nicolas.

**Comment by jackMort** [comment]
`ctrl+y`

**Reply by nbossard** [reply]
This works, thanks, Nicolas
